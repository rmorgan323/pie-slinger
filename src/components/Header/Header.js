import React from 'react';
import './Header.css';

const Header = () => {

  return (
    <div className="Header">
      <h1>pie<span className="logo-span">slinger</span></h1>
    </div>
  );
};

export default Header;