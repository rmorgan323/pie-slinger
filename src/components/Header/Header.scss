@import '../../variables.scss';

.Header {
  background-color: $brown3;
  border-bottom: 6px solid $brown2;
  margin-bottom: 36px;
  padding: 10px 20px;

  h1 {
    color: #fff;
  }

  .logo-span {
    color: $red2;
  }
}