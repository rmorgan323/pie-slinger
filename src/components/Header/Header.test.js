import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

it('should match the snapshot', () => {
  const renderedApp = shallow(<Header />);
  expect(renderedApp).toMatchSnapshot();
});