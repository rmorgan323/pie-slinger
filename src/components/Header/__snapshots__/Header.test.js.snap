// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        pie
        <span
                className="logo-span"
        >
                slinger
        </span>
</h1>,
      "className": "Header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "pie",
          <span
            className="logo-span"
>
            slinger
</span>,
        ],
      },
      "ref": null,
      "rendered": Array [
        "pie",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "slinger",
            "className": "logo-span",
          },
          "ref": null,
          "rendered": "slinger",
          "type": "span",
        },
      ],
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          pie
          <span
                    className="logo-span"
          >
                    slinger
          </span>
</h1>,
        "className": "Header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "pie",
            <span
              className="logo-span"
>
              slinger
</span>,
          ],
        },
        "ref": null,
        "rendered": Array [
          "pie",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "slinger",
              "className": "logo-span",
            },
            "ref": null,
            "rendered": "slinger",
            "type": "span",
          },
        ],
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
